src/lexicon.js

```diff
@@ -0,0 +1,33 @@
+
+const lexicon = function (fetch, baseUrl, token) {
+	const api = {
+		lexica: {
+			listFromTraining (page, language, trainingId) {
+				return api.fetch(`lexica/?page=${page}&language=${language}&training_id=${trainingId}`)
+			},
+			create (entryJson) {
+				return api.fetch('lexica/', 'POST', entryJson)
+			},
+			delete (id) {
+				return api.fetch(`lexica/${id}/`, 'DELETE')
+			},
+			getLexiconEntry (id) {
```

Comment:
What is the difference between a lexicon and a lexicon entry? URL suggests a whole lexicon.